### ISSUE TITLE
How to map taxon pairs in phenoscape homology file

### ISSUE THREAD

**Issue by cmungall** [BgeeDB/anatomical-similarity-annotations#13]
We are preparing to submit a set of homology annotations created by the Phenoscape project (it may be useful to have a github label for external submissions or similar?)

The phenoscape associations hold between `(structure, taxon)` tuples. E.g `(limb,tetrapod)<->(fin<->teleost)`.

These should be mapped to a single taxon for submission with phenoscape. See

https://github.com/BgeeDB/anatomical-similarity-annotations/wiki/Similarity-annotations#entity-column-3

Here the MRCA of the two taxa can be used. However, I will verify each entry to ensure the semantics are correct prior to submission

**Comment by fbastian** [comment]
Yep, in the Bgee annotations, this would be represented in 3 annotations: 
- limb, tetrapod
- fin, teleost
- limb|fin, Euteleostomi
